source/lib-intro.tex

```diff
@@ -447,6 +469,23 @@
 must be implemented.\footnote{Although in some cases the code given is
 unambiguously the optimum implementation.}
 
+\pnum
+Required operations of any concept defined in this document need not be
+total functions; that is, some arguments to a required operation may
+result in the required semantics failing to be satisfied.
+\begin{example}
+The required \tcode{<} operator of the \tcode{StrictTotallyOrdered}
+concept\iref{concepts.lib.compare.stricttotallyordered} does not meet the
+semantic requirements of that concept when operating on NaNs.
+\end{example}
+This does not affect whether a type satisfies the concept.
+
+\pnum
+A declaration may explicitly impose requirements through its associated
+constraints\iref{temp.constr.decl}. When the associated constraints refer to a
+concept\iref{temp.concept}, additional semantic requirements are imposed on the
+use of the declaration.
```

Comment:
"additional" is great, but which ones in particular? Maybe the semantic requirements explained in the subclause that defines the concept?

Reply:
How about "When the associated constraints refer to a concept\iref{temp.concept}, the semantic constraints specified for that concept are additionally imposed on the use of the declaration."?

Reply:
Yes, seems to be an improvement. (Make sure to keep those rephrasings separate from the original commit with the as-moved paper.)